src/noc/containers/noc-home/noc-home.component.html

```diff
@@ -0,0 +1,39 @@
+<exui-app-header></exui-app-header>
+<div class="govuk-width-container">
+  <div class="page">
+    <main id="content" role="main">
+      <div class="govuk-grid-row">
+        <exui-noc-navigation (eventTrigger)="onNavEvent($event)">
+          <div class="govuk-main-wrapper">
+            <exui-noc-case-ref
```

Comment:
Can only one of these be visible at a time? If so, a switch would be better. If not, ignore this comment.

Reply:
yes it can be visible at a time by state management